core/frontend/src/Viewport.ts

```diff
@@ -104,7 +104,7 @@ export enum DepthPointSource {
   Map,
 }
 
-/** @alpha Options to control behavior of [[Viewport.pickDepthPoint]]. */
+/** @public Options to control behavior of [[Viewport.pickDepthPoint]]. */

```

Comment:
Not sure if this formatting will show up correctly in the docs.  You will probably need to move the actual description to the first line and have the release tag on a second line.

Reply:
Changed just to be safe